source/agg-pipeline-builder/pipeline-custom-collation.txt

```diff
@@ -0,0 +1,54 @@
+.. _pipeline-custom-collation:
+
+========================
+Specify Custom Collation
+========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use custom collation to specify language-specific rules for string comparison, 
+such as rules for letter case and accent marks.
+
+About This Task 
+---------------
+
+When entering a collation document, the ``locale`` field is mandatory. Default 
+collation parameter values vary depending on which locale you specify. To learn 
+more about supported languages and locales, see 
+:manual:`Collation Locales and Default Parameters </reference/collation-locales-defaults>`.
+
+
+Procedure
+---------
+
+.. procedure:: 
+   :style: normal
+   
+   .. step:: Click :guilabel:`More Options`.
+    
+      In the top right corner of the pipeline builder, click 
+      :guilabel:`More Options`.
+      
+      .. figure:: /images/compass/options-collation.png
+         :alt: More Options
+         
+   .. step:: Enter your collation document.
+
+      Next to the :guilabel:`Collation` field, enter your 
+      :manual:`collation document </reference/collation/>`.
+
+      After you enter your collation document, the aggregation pipeline builder 
+      considers the language-specific rules that you specified in your document.
```

Comment:
[suggestion]

Can we add an example collation document here? Might be nice just to help users make sure they have the correct syntax.

Reply:
Adding!